### DISCUSSION TITLE
Plot_contour, how to see other parameters & trial value?

### DISCUSSION THREAD

**Discussion by blazespinnaker** [optuna/optuna#3840] [Q&A]
I think I must be missing something.  Perhaps it's my chromebook?  The plot contour visualization is extremely informative, but when I click on the individual plots, I can't see the full information regarding the trial(s) it represents.  Is that because it's not implemented that way (perhaps because a tuple of values could be associated with many trials), or is it because I'm missing an option somewhere?

Perhaps we could pass a function which accepts a list of trials and outputs a string to be included on the plot?   This would be extremely useful.  As it is, I have to wade through other data sources / graphs to see this information. Doable, but it'd be fantastic if it was all in one place.

**Comment by nzw0301** [comment]
Sorry, I could not understand the question. Could you elaborate on your question a little more?

**Comment by blazespinnaker** [comment]
I was trying to use plot_contour to visually inspected and browsing trials rather than say using a trial dataframe.  I had hoped I could see the full information about a trial which is plotted by mouse selecting it in some manner, but perhaps this is not a common use case.  

I am using the trial dataframe more now and in someways it's probably better, so perhaps not an issue.

**Comment by blazespinnaker** [comment]
Looking at the multi objective viz, I see that indeed it does show parameters and trial values.
![Screenshot 2022-08-18 5 15 31 PM](https://user-images.githubusercontent.com/2188809/185516230-72e069fc-7285-4258-a07b-fd56cbac40c5.png)

https://optuna.readthedocs.io/en/stable/tutorial/20_recipes/002_multi_objective.html

Executing that code results in similar results, however these aren't showing up for me on my studies (they are backed by mysql) - just the trial number.
